realsense/scene_perception/win/scene_perception_object.cc

```diff
@@ -820,54 +749,44 @@ void ScenePerceptionObject::OnGetVertices(
 
   sensemanager_thread_.message_loop()->PostTask(
       FROM_HERE,
-      base::Bind(&ScenePerceptionObject::DoCopyVertices,
+      base::Bind(&ScenePerceptionObject::DoGetVertices,
                  base::Unretained(this),
                  base::Passed(&info)));
 }
 
-void ScenePerceptionObject::DoCopyVertices(
+void ScenePerceptionObject::DoGetVertices(
```

Comment:
I prefer you merge <code>DoGetVertices</code> and <code>DoGetNormals</code> as one common function, such as <code>DoGetVerticesOrNormal</code> with an additional parameter to indicate it's vertices data or normals data since their function body are almost same. Meanwhile you can use a unify binary message name, such as <code>binary_message_data_</code> to instead of <code>normals_message</code> and <code>vertices_message</code>